### PR TITLE
Jenliz

### PR DESCRIPTION
**Pull request by jenniferbacon01**
User can edit profile info

### CODE REVIEW

**Review by LizzieHard — APPROVED** [review]

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-  def show
+  def edit
     @user = current_user
   end
 
+  def update
+    @user = current_user
+    @user.update(post_params)
+    redirect_to action: :show
+ end
```
Comment (maryrosecook) [inline app/controllers/users_controller.rb:10]: Use idiomatic indentation.

File: db/schema.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+
```
Comment (maryrosecook) [inline db/schema.rb:13]: Watch out for extra random whitespace.

File: db/schema.rb [hunk 3]
```diff
@@ -27,6 +28,9 @@
     t.string "message"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "user"
```
Comment (maryrosecook) [inline db/schema.rb:31]: Not sure what this field is for.

**Review by maryrosecook — COMMENTED** [review]
Here you go.  Let me know if you have any questions!

**Pull request closed without merging**